classes/log/SubmissionEmailLogDAO.php

```diff
@@ -69,27 +69,26 @@ public function getBySubmissionId($submissionId)
     /**
      * Get submission email log entries by submission ID, event type and sender ID
      *
-     * @param int $submissionId
      * @param int $eventType SubmissionEmailLogEntry::SUBMISSION_EMAIL_*
      * @param int $senderId Return only emails sent by this user.
      *
      * @return DAOResultFactory<SubmissionEmailLogEntry>
      */
-    public function getBySenderId($submissionId, $eventType, $senderId)
+    public function getBySenderId(int $submissionId, int $eventType, int $senderId): DAOResultFactory
```

Comment:
These changes are probably going to conflict with @taslangraham's work over on https://github.com/pkp/pkp-lib/issues/10129. I'd suggest leaving the cleanups on `SubmissionEmailLogDAO` et al out of this PR.

Reply:
The PR was merged, so I'll just do a rebase later.